crates/integrations/datafusion/src/physical_plan/expr_to_predicate.rs

```diff
@@ -403,4 +452,55 @@ mod tests {
             Reference::new("ts").greater_than_or_equal_to(Datum::string("2023-01-05T00:00:00"));
         assert_eq!(predicate, expected_predicate);
     }
+
+    #[test]
+    fn test_to_timestamp_comparison_creates_predicate() {
+        let sql = "ts >= timestamp '2023-01-05T00:00:00'";
+        let predicate = convert_to_iceberg_predicate(sql).unwrap();
+        let expected_predicate =
+            Reference::new("ts").greater_than_or_equal_to(Datum::string("2023-01-05T00:00:00"));
+        assert_eq!(predicate, expected_predicate);
+    }
```

Comment:
This test looks the same as `test_predicate_conversion_with_cast`

Reply:
removed